IdentityCore/src/cache/metadata/accountMetadata/MSIDAccountMetadataCacheItem.h

```diff
@@ -30,6 +30,8 @@
 @property (nonatomic, readonly) NSString *clientId;
 @property (nonatomic, readonly) NSDictionary *internalMap;
 
+@property (nonatomic, readonly) BOOL isSignedOut;
```

Comment:
should this be an enum with different states for future extensibility?

Reply:
good idea, changed :)